src/werkzeug/middleware/shared_data.py

```diff
@@ -87,6 +88,10 @@ class SharedDataMiddleware:
     :param cache_timeout: the cache timeout in seconds for the headers.
     :param fallback_mimetype: The fallback mimetype for unknown files.
 
+    .. versionchanged:: 3.1.0
```

Comment:
This can be removed, I haven't been calling out typing fixes with `versionchanged`.